pkgs/applications/networking/instant-messengers/pidgin/default.nix

```diff
@@ -94,6 +94,6 @@ let unwrapped = stdenv.mkDerivation rec {
 
 in if plugins == [] then unwrapped
     else import ./wrapper.nix {
-      inherit makeWrapper symlinkJoin plugins;
+      inherit lib makeWrapper symlinkJoin plugins;
```

Comment:
I think we can drop the inherit if we use callPackage.

Reply:
Sounds good. Added `callPackage`. `inherit` is left for `plugins`.